src/status_im2/contexts/quo_preview/wallet/transaction_progress.cljs

```diff
@@ -0,0 +1,155 @@
+(ns status-im2.contexts.quo-preview.wallet.transaction-progress
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.common.resources :as resources]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:type :text
+    :key  :title}
+   {:type :text
+    :key  :tag-name}
+   {:type :text
+    :key  :epoch-number-mainnet}
+   {:type :text
+    :key  :epoch-number-optimism}
+   {:type :text
+    :key  :epoch-number-arbitrum}
+   {:type :text
+    :key  :tag-number}
+   {:type :text
+    :key  :optimism-progress-percentage}
+   {:type :text
+    :key  :arbitrum-progress-percentage}
+   {:type    :select
+    :key     :network
+    :options [{:key :mainnet}
+              {:key :optimism}
+              {:key :arbitrum}
+              {:key :optimism-arbitrum}]}
+   {:type    :select
+    :key     :state-mainnet
+    :options [{:key :pending}
+              {:key :sending}
+              {:key :confirmed}
+              {:key :finalising}
+              {:key :finalized}
+              {:key :error}]}
+   {:type    :select
+    :key     :state-optimism
+    :options [{:key :pending}
+              {:key :sending}
+              {:key :confirmed}
+              {:key :finalising}
+              {:key :finalized}
+              {:key :error}]}
+   {:type    :select
+    :key     :state-arbitrum
+    :options [{:key :pending}
+              {:key :sending}
+              {:key :confirmed}
+              {:key :finalising}
+              {:key :finalized}
+              {:key :error}]}
+   (preview/customization-color-option)])
+
+(def total-box 85)
+(def counter (reagent/atom 0))
```

Comment:
given that this is a preview screen I think it's okay to add this as a global ratom, but this is generally something we should try avoid unless absolutely necessary 👍

Reply:
Yes, I'm aware of that, but here it's easier to do so since it's a preview screen